Tasks/DotNetCoreInstallerV1/task.json

```diff
@@ -1,9 +1,9 @@
 {
     "id": "B0CE7256-7898-45D3-9CB5-176B752BFEA6",
     "name": "DotNetCoreInstaller",
-    "friendlyName": ".NET Core SDK/Runtime Installer",
-    "description": "Acquires a version of the .NET Core SDK/Runtime from the internet or the local cache and adds it to the PATH. Use this task to change the version of .NET Core used in subsequent tasks.",
-    "helpUrl": "https://aka.ms/AA4ihfi",
+    "friendlyName": ".NET Core sdk/runtime installer",
+    "description": "Acquire a specific version of the .NET Core SDK from the internet or local cache and add it to the PATH",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer",
```

Comment:
This still points to v0 documentation. Should we update the same page or add another one with the update?

Reply:
@sachinma I'm glad you noticed that.  Until a better pattern is established, the content team's current approach is to update the same doc page to the latest info about the task.

Reply:
Ok David. I will work with @pulkitaggarwl  to get the .Net core documentation updated to the latest version.